docs/admin/logging/fluentd.md

```diff
@@ -79,20 +78,6 @@ the log tag format.
 
 The `labels` and `env` options each take a comma-separated list of keys. If there is collision between `label` and `env` keys, the value of the `env` takes precedence. Both options add additional fields to the extra attributes of a logging message.
 
-### fail-on-startup-error
```

Comment:
@pcarrier can we explain how `fluentd-async-connect` work here? I think that's enough to have the docs ready. I recommend you to rebase master at the same time :wink: